css/tailwind.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+	purge: [
+		'../templates/**/*.html'
```

Comment:
when `NODE_ENV=production`, `tailwind-cli` will crawl the `html` files in `templates` and "purge" any unused styles.